client/server/server.go

```diff
@@ -386,7 +386,7 @@ func (s *Server) Up(callerCtx context.Context, _ *proto.UpRequest) (*proto.UpRes
 	}
 
 	if s.statusRecorder == nil {
-		s.statusRecorder = peer.NewRecorder()
+		s.statusRecorder = peer.NewRecorder(s.config.ManagementURL.String())
```

Comment:
To avoid management URL caching, we need to clean the `s.statusRecorder ` on `server.Down` calls. So when the user wants to update the management URL with the `netbird up -m https://newManager` the URL will reflect the change.